examples/src/features.js

```diff
@@ -0,0 +1,4 @@
+// feature config for feature flags
+export default {
+    tasks: true,
```

Comment:
Can we have this match the existing prop style like has*, maybe `hasNewTasks`

Reply:
can just remove for now, I'll update the tasks PR if we want to use that convention